Formula/bazel.rb

```diff
@@ -16,6 +16,8 @@ class Bazel < Formula
   desc "Fast, scalable, multi-language and extensible build system"
   homepage "https://bazel.build/"
   url "https://releases.bazel.build/3.1.0/release/bazel-3.1.0-installer-darwin-x86_64.sh", :using => :nounzip
+  # Mirror for users in China
+  mirror "https://github.com/bazelbuild/bazel/releases/download/3.1.0/bazel-3.1.0-installer-darwin-x86_64.sh"
```

Comment:
Does the .rb file format allow for variables? It'd be nice to only need to update one place when the version increments.

Reply:
I haven't tested it, but according to the docs this should work:

```ruby
version = "3.1.0"

url "https://releases.baze.build/#{version}/release/bazel-#{version}-installer-darwin-x86_64.sh", :using => :nounzip
mirror "..."
```

Reply:
Oh, wait. This is inside a Ruby class - then I'm not sure how it would work :/

Reply:
I believe you can use ALL_CAPS for global variables.